### DISCUSSION TITLE
Force Author Images to re-download

### DISCUSSION THREAD

**Discussion by BDaddyG** [advplyr/audiobookshelf#1915] [General]
After a failed upgrade to v2.3.0, I restored v2.2.23 using a backup file to clean /config and /metadata folders.

When I look at Authors, the images are missing (as expected, as I understand backups don't include the image files).

When I perform a Quick Match, the message "No Updates for <author> were made." is displayed, and the image file is not updated.  I have to open the author, delete the current broken image, then the image file is downloaded again.

Is there a way for force download all the images again?

**Comment by advplyr** [comment]
Backups do include the image files. If you extract the backup file (it is just a zip file with extension `.audiobookshelf`) you will see the `metadata-authors` folder. Put the contents of that folder in `/metadata/authors`

**Reply by BDaddyG** [reply]
Yes, after looking in the backup file I do see files in the `/metadata/authors` folder.

I wonder why the restore process didn't restore the files to `/metadata/authors`?

I did copy the .JPGs from a backup folder and the images are displayed without having to delete / download again.

**Comment by skyzuma** [comment]
maybe same or similar issue here ... the migration / upgrade to 2.3.0 was successful but the images of the authors was gone ... ive forced to redownload the images and now all there ...
